components/src/dynamo/sglang/benchmark.py

```
@@ -0,0 +1,173 @@
+import json
+import logging
+import os
+import time
+from pathlib import Path
+from typing import Any, Optional
+
+from dynamo.sglang.capacity import local_dp_rank_bounds
+
+logger = logging.getLogger(__name__)
+
+_DEFAULT_BENCHMARK_OUTPUT_PATH = "/tmp/benchmark_results.json"
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/benchmark_results.json"

(S108)